rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/OpenehrEhrStatusController.java

```diff
@@ -182,10 +182,11 @@ public OpenehrEhrStatusController(EhrService ehrService) {
 
         List<String> headerList = Arrays.asList(CONTENT_TYPE, LOCATION, ETAG, LAST_MODIFIED);   // whatever is required by REST spec
         Optional<InternalResponse<EhrStatusResponseData>> respData;   // variable to overload with more specific object if requested
+        UUID statusUid = UUID.fromString(status.getUid().getValue().split("::")[0]);
```

Comment:
@jakesmolka note in the spec, the OBJECT_VERSION_ID is

uid: UID
system_id: UID
version_tree_id: VERSION_TREE_ID

UID is abstract, could be UUID, OID or INTERNET_ID

Here status.getUid() should be an OBJECT_VERSION_ID, so the first part is a UID, not a UUID. For instance the uid could be "2.16.234.234.4353.364" (an OID).

If we are not going to support OIDs I guess we need to document it.

REF: https://specifications.openehr.org/releases/BASE/latest/base_types.html#_overview_4